### DISCUSSION TITLE
Support search for Layers as well as Assets - don't repeat the mistake of Figma!  :-)

### DISCUSSION THREAD

**Discussion by steveharman** [uxbox/uxbox#1388] [Ideas]
Hi!

My biggest "Huh?" with Figma which drives me crazy on a daily basis, appears to be being repeated with Penpot!  :-)

Obviously being in beta I appreciate that it's early days but please can I put in a huge request to support a search feature filter across **layers** in the same way as well already is for assets?  

To my mind it just doesn't add up that searching assets would be considered really useful, but nobody would want to search their _layers_ ?

Working on projects with perhaps 50+ "screens"on a page it was always a dream in `Sketch` to simply enter a search term into the layers search box and see all the matches filtered beneath. I could then click a layer and use the keyboard shortcut to zoom-to-selection and get right to where I needed to be.   Figma has this exact search feature _for assets_, as does Penpot - but neither seem to offer it **for layers** and it feels like a big miss.    

Sketch made it incredibly easy to find all those hard to reach layers with a simple search.  If I wanted to find all the instances of "Group" where I'd been lazy and not given my groups a better name - simply filter layers on "Group" and there they were.  It was a feature I used multiple times every day until moving to Figma.

Many thanks,

Steve

<img width="793" alt="search" src="https://user-images.githubusercontent.com/12557929/144286423-1901da7f-2e19-48a2-b823-1b4eb743604d.png">

**Comment by hirunatan** [comment]
@steveharman you are true! This would be useful. I've created an issue for this. We'll do it when it's possible: https://tree.taiga.io/project/penpot/issue/2354

**Reply by diacritica** [reply]
@steveharman good news, it was promoted to user story and it's now being defined. It even deserves some time during our dailies. See https://tree.taiga.io/project/penpot/us/2564
